style.css

```diff
@@ -85,3 +97,154 @@ body {
 .icon-img li {
   list-style-type: none;
 }
+
+.hold-image {
+  height: 300px;
+  width: 90%;
+  border-radius: 8px;
+}
+
+.lists {
+  display: flex;
+  justify-content: space-between;
+  width: 80%;
+}
+
+.lists-item {
+  color: #7a869a;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+}
+
+.lang-container {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  padding-left: 0%;
+  list-style-type: none;
+  justify-content: flex-start;
+}
+
+.lang-item {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 5px 10px;
+  margin: 0 2px;
+  font-size: 14px;
+  color: #396df2;
+}
+
+button {
+  cursor: pointer;
+}
+
+button:enabled {
+  color: #6070ff;
+  background-color: #fff;
+  border-color: #6070ff;
+}
+
+button:hover {
+  color: #fff;
+  background-color: #6070ff;
+}
+
+button:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+button:disabled {
+  color: #5e6c84;
+  border-color: #5e6c84;
+  background-color: #fff;
+}
+
+.click-btn {
+  font-size: 19px;
+  color: #396df2;
+  padding: 13px;
+  background-color: transparent;
+  margin-top: 2rem;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+}
+
+.card-contain {
+  border: 1px solid #7a869a;
+  width: 90%;
+  margin: 12px auto;
+  border-radius: 16px;
+  background-color: #fff;
+}
+
+.btn {
+  height: 10px;
+  padding: 9px;
+}
+
+.card-contain li {
+  list-style-type: none;
+}
+
+.about-container {
+  width: 90%;
+  margin: 0 auto;
+  overflow: hidden;
```

Comment:
- [ ] I would kindly suggest you stick to the Figma design as much as possible. Currently, the details in your project do not look exactly like what we have in the Figma design.

  - The Image in the project card is not fitting well with the card as we have it in the Figma design
  - The `back end dev` is not aligning properly with what we have in your Figma design

Your design 👇 

![rol](https://user-images.githubusercontent.com/78634841/212288223-4e9b3e6c-80b5-4bfb-9282-ef5ddb53b717.png)

Figma design 👇 

![card real](https://user-images.githubusercontent.com/78634841/212288586-c58bca2c-246d-41af-b516-cf8b5bf1aa3c.png)